### DISCUSSION TITLE
Commit background color is white in dark mode

### DISCUSSION THREAD

**Discussion by okyanusoz** [community/community#5010] [Other Feature Feedback, Questions, & Ideas] [answered]
When I open a commit in dark mode, the background color doesn't match. This looks weird. See attached image.

![Image](https://user-images.githubusercontent.com/46757266/128923486-3fc99639-d327-44c2-9b73-3c935784ba24.jpg)

**Comment by gaknoll** [comment, marked as answer]
Thanks @okyanusoz for reporting this! A couple questions for you to help us diagnose what could be causing this:
* Are you using any GitHub related browser extensions? This has been the number one cause of weird UI issues like this in the past. 
* Potentially independent or related to point above, but have you opted in to the [Dark High Contrast theme beta](https://github.blog/changelog/2021-06-28-dark-high-contrast-theme-beta-for-github-com/)? Part of the dark high contrast release is an update to our color system which could impact GitHub when using browser extensions (at least until the browser extension ships a fix after we GA).

**Reply by Shamus03** [reply]
I see the same thing.  I have not opted into the beta.

**Reply by gaknoll** [reply]
I took this back to the team as well and it was not caused by any of the above. We are in the process of shipping a fix! Thanks again for reporting this bug 🙏🏽

**Reply by okyanusoz** [reply]
You're welcome!
